src/Processors/QueryPlan/FilterStep.cpp

```diff
@@ -42,4 +43,19 @@ void FilterStep::transformPipeline(QueryPipeline & pipeline)
     });
 }
 
+void FilterStep::describeActions(FormatSettings & settings) const
+{
+    String prefix(settings.offset, ' ');
+    settings.out << prefix << "Filter column: " << filter_column_name << '\n';
+
+    bool first = true;
+    for (const auto & action : expression->getActions())
+    {
+        settings.out << prefix << (first ? "Actions: "
```

Comment:
It can be considered normal.
The user can see how data was filtered and the fact that data is being filtered but they cannot see the data that was filtered out.

Reply:
Ok, but then what is the purpose of restricting access to the `system.row_policies`/`show row policies` ?

Reply:
It prevents to show policies for all other users?

Reply:
You are right, ok then